l1-contracts/remappings.txt

```diff
@@ -4,6 +4,7 @@ ds-test/=lib/forge-std/lib/ds-test/src/
 eth-gas-reporter/=node_modules/eth-gas-reporter/
 forge-std/=lib/forge-std/src/
 hardhat/=node_modules/hardhat/
-solpp/=cache/solpp-generated-contracts/
 murky/=lib/murky/src/
 foundry-test/=test/foundry/
+contracts/=contracts/
```

Comment:
Is this really neeeded?

Reply:
Looks like my solidity IDE extension was throwing an error but foundry compiles just fine without it, so I removed it.